views/options/profile.jade

```diff
@@ -213,6 +213,13 @@ script(id='partials/options.profile.profile.html', type='text/ng-template')
         button.btn.btn-default(ng-click='_editing.profile = true', ng-show='!_editing.profile')= env.t('edit')
         h4=env.t('displayName')
         span(ng-show='profile.profile.name') {{profile.profile.name}}
+        p
+            small.muted
+                =env.t('displayNameDescription1')
+                |&nbsp;
+                a(href='https://habitrpg.com/#/options/settings/settings')=env.t('displayNameDescription2')
```

Comment:
Can we make this a relative link instead? `href='/#/options/settings/settings'`, that way the link will navigate to the correct page on local copies of the app as well as the webapp.